src/Zikula/CoreBundle/Command/GenerateTestUsersCommand.php

```diff
@@ -132,7 +133,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->verified = in_array((int) $input->getOption('verified'), [0, 1, 2]) ? (int) $input->getOption('verified') : 1;
         $regDate = $input->getOption('regdate') ?? $this->nowUTC;
         $this->range = is_string($regDate) && '>' === $regDate[0];
-        $this->regDate = $this->range ? mb_substr($regDate, 1) : $regDate;
+        $this->regDate = $this->range ? (string)s($regDate)->slice(1) : $regDate;
```

Comment:
space after cast or no? here you do not have a space, later you do have a space.

Reply:
Spacing for casts were not consistent before already. Improved that in 7a1aa6ab9130a1ffde0c5c997d5fe5c005ea7cac